assets/scss/layouts/products/_productList.scss

```diff
@@ -23,6 +23,21 @@
 // -----------------------------------------------------------------------------
 
 .listItem {
+    &:focus-within {
+        .listItem-figureBody {
+            @include breakpoint("large") {
+                opacity: 1;
+            }
+        }
+    }
+
+    &.focus-within {
+        .listItem-figureBody {
+            @include breakpoint("large") {
+                opacity: 1;
+            }
+        }
+    }
```

Comment:
Could we change it to `&:focus-within, &.focus-within {`?

Reply:
yes

Reply:
Suggested approach does not work in Edge and IE